themes/default/content/registry/packages/kubernetes/how-to-guides/managing-resources-with-server-side-apply.md

```diff
@@ -15,6 +15,12 @@ The [v3.20.0 release](https://github.com/pulumi/pulumi-kubernetes/releases/tag/v
 
 This guide will show you how to enable SSA support and use these features to manage Kubernetes resources.
 
+## Prerequisites
+
+Server-Side Apply support requires `patch` permission on the Kubernetes cluster for all operations, including previews.
+Previews will not modify the state of your cluster, but use the `dry-run` mode of the `patch` operation to compute a
+preview diff. See the [upstream Kubernetes docs](https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run-authorization) for additional information about this requirement.
```

Comment:
```suggestion
Previews will use the `dry-run` mode of the `patch` operation to compute a preview diff, and not modify the 
state of your cluster. See the [upstream Kubernetes docs](https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run-authorization) for additional information about this requirement.
```

The "but" threw me off when reading it -- I reckon this is a little clearer for avoiding the "but", wdyt?

If it's not getting too verbose, you could pull in the fact that the Kubernetes API server enforces no mutations on dry-runs.